tools/slack_tools/collect_prs.py

```diff
@@ -0,0 +1,192 @@
+# Script for collecting PRs in need of review, and informing maintainers via
+# slack
+
+from __future__ import print_function
+
+import datetime
+import os
+import sys
+
+import github
+
+try:
+    input = raw_input  # Python 2
+except NameError:
```

Comment:
this shouldnt be necessary - just assume python3 - or depending on how its triggered use a python3 shebang line